router/mux/engine.go

```diff
@@ -48,3 +48,13 @@ func (i *HTTPErrorInterceptor) WriteHeader(code int) {
 	})
 	i.ResponseWriter.WriteHeader(code)
 }
+
+type Handler struct {
+	*http.ServeMux
+}
+
+func NewHandler() *Handler { return &Handler{http.NewServeMux()} }
+
+func (e *Handler) Handle(pattern, method string, handler http.Handler) {
+	e.ServeMux.Handle(pattern, handler)
```

Comment:
since this is ignoring the method, it will panic just as the current implementation when two methods are registered at the same pattern